projects/plugins/jetpack/tests/php/core-api/wpcom-endpoints/test-class-wpcom-rest-api-v2-endpoint-admin-menu.php

```diff
@@ -92,7 +92,7 @@ public function test_parent_menu_item_always_exists() {
 		$request  = wp_rest_request( Requests::GET, '/wpcom/v2/admin-menu' );
 		$response = $this->server->dispatch( $request );
 
-		$menu      = wp_list_filter( $response->get_data(), array( 'title' => 'Settings' ) );
+		$menu      = wp_list_filter( $response->get_data(), array( 'title' => 'Jetpack' ) );
```

Comment:
The Settings menu no longer returns submenu items for Jetpack sites, so I had to update this test to user another menu with submenus.